### DISCUSSION TITLE
2018/09/14/infusion

### DISCUSSION THREAD

**Discussion by giscus** [wanghenshui/wanghenshui.github.io#93] [Q&A]
# 2018/09/14/infusion

https://wanghenshui.github.io/2018/09/14/infusion.html

**Comment by zhanxiangyu** [comment]
有没有infusion软件资源？

**Reply by wanghenshui** [reply]
这玩意一般都是公司买的

**Reply by zhanxiangyu** [reply]
我看官网都挂了，购买渠道都没有

**Comment by wanghenshui** [comment]
那应该就死了，别研究了。随便一个代码分析工具都差不多，clangtidy之类的












很水
***@***.***


---Original---
From: ***@***.***&gt;
Date: Tue, Jul 11, 2023 11:20 AM
To: ***@***.***&gt;;
Cc: "Ted ***@***.***&gt;;"State ***@***.***&gt;;
Subject: Re: [wanghenshui/wanghenshui.github.io] 2018/09/14/infusion(Discussion #93)




 
我看官网都挂了，购买渠道都没有

**Reply by zhanxiangyu** [reply]
感谢指导
